plugin/trino-redis/src/test/java/io/trino/plugin/redis/TestRedisConnectorConfig.java

```diff
@@ -45,12 +48,20 @@ public void testDefaults()
                 .setRedisPassword(null)
                 .setRedisScanCount(100)
                 .setRedisMaxKeysPerFetch(100)
-                .setHideInternalColumns(true));
+                .setHideInternalColumns(true)
+                .setTlsEnabled(false)
+                .setKeystorePath(null)
+                .setKeystorePassword(null)
+                .setTruststorePath(null)
+                .setTruststorePassword(null));
     }
 
     @Test
-    public void testExplicitPropertyMappings()
+    public void testExplicitPropertyMappings() throws IOException
```

Comment:
`throws IOException` on the new line. Throwing `Exception` is fine in test methods.